src/services/account/services/SupportJWTService.js

```diff
@@ -125,16 +123,16 @@ class SupportJWTService {
 	executeInfo(currentUserId, userId) {
 		const minutes = this.expiredOffset / (60 * 1000);
 		// eslint-disable-next-line max-len
-		logger.info(`[support][jwt] The support employee with the Id ${currentUserId} has created  a short live JWT for the user with the Id ${userId}. The JWT expires expires in ${minutes} minutes`);
+		logger.info(
+			`[support][jwt] The support employee with the Id ${currentUserId} has created  a short live JWT for the user with the Id ${userId}. The JWT expires expires in ${minutes} minutes`
```

Comment:
Codacy found an issue: [This line has a length of 190. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)